### PR TITLE
sys-libs/ncurses:5 → sys-libs/ncurses-compat

### PR DESCRIPTION
**Pull request by mgorny**
Following other packages, time has come for ncurses — which is probably biggest offender in the category of MissingSlotDeps.

For a start, a CI run to determine which deps need updating (and if CI catches all I've found correctly).

### CODE REVIEW

File: games-roguelike/adom/adom-1.1.1-r1.ebuild [hunk 1]
```diff
@@ -15,9 +15,7 @@ IUSE=""
 RESTRICT="strip" #bug #137340
 QA_FLAGS_IGNORED="${GAMES_PREFIX_OPT:1}/bin/adom"
 
-RDEPEND="|| (
-	>=sys-libs/ncurses-5.9-r3:0/5[abi_x86_32(-)]
-	>=sys-libs/ncurses-5.9-r3:5/5[abi_x86_32(-)] )"
+RDEPEND="sys-libs/ncurses-compat:5"
```
Comment (chewi) [inline games-roguelike/adom/adom-1.1.1-r1.ebuild]: You lost the `abi_x86_32` flag here.

**Comment by ryao** [comment]
My approval is just for app-emulation/crossover-bin. I have not reviewed the other ebuilds as I do not maintain them.

File: games-roguelike/adom/adom-1.1.1-r1.ebuild (hunk 1 above)
Comment (mgorny) [reply games-roguelike/adom/adom-1.1.1-r1.ebuild]: Fixed.

**Comment by mgorny** [comment]
Could someone from @gentoo/base-system ack this?

**Comment by Whissi** [comment]
We are currently discussing this PR (please note that primary maintainer is currently on vacation) and will provide an update shortly.

**Comment by mgorny** [comment]
> Approval from base-system project.
> 
> @mgorny Next time you do such change please talk to us **before** starting to do anything. We would really appreciate **any** communication. Thank you.

The RFC was sent back in January, if I recall correctly. It's not my fault that you didn't read it or forgot about it.

**Pull request closed without merging**